public/scripts/application.js

```diff
@@ -1,4 +1,4 @@
-head.ready(function() {
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
Why not use the jQuery `$(function() { ... });`?

Reply:
Oh, i've never used jquery. In fact, jquery is only used in one call for the entire project right now. I could rewrite it and remove jquery entirely from the project. It would mean that you don't have to update it when new versions come out, and it would make the project slightly more compact and secure? Or we can stick with jquery. Your call :)